File: storage/data_original/at_coder/arc020/A/4224942.java
import java.util.*;

public class Main {
	public static void main (String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = Math.abs(sc.nextInt());
		int b = Math.abs(sc.nextInt());
		if (a < b) {
			System.out.println("Ant");
		} else if (a > b) {
			System.out.println("Bug");
		} else {
			System.out.println("Draw");
		}
	}
}

File: data/data_transpiled/at_coder/arc020/A/4224942.py
def import sys
class Main ( object ) :
    def __init__ ( self ) :
        self.sc = sys.stdin
        a = abs ( self.sc.read ( ) )
        b = abs ( self.sc.read ( ) )
        if a < b :
            print ( "Ant" )
        elif a > b :
            print ( "Bug" )
        else :
            print ( "Draw" )
